docs/01-overview/main-areas/observability/obsv-04-telemetry-in-kyma.md

```diff
@@ -147,6 +147,8 @@ spec:
         - fluent-bit
 ```
 
+It is recommended to exclude the logs of the Fluent Bit container to avoid problems with recursive logs in case Fluent Bit will print an error per processed log line which then gets collected and processed again.
```

Comment:
```suggestion
It might happen that Fluent Bit prints an error per processed log line that is then collected and re-processed.
To avoid problems with such recursive logs, it is recommended that you exclude the logs of the Fluent Bit container. 
```

This sentence is very long. How about this instead?